### PR TITLE
Feature/personal tasks

### LINKED ISSUE #1: Feature: Personal todos CRUD needed
- Write CRUD Operations for personal todos
- Write middleware for authorization

### PR DESCRIPTION
**Pull request by omkarvaigankar8**
Personal Tasks CRUD created
resolves #1

### CODE REVIEW

File: src/config/zodSchema.config.ts
Comment (arpesh28) [inline src/config/zodSchema.config.ts:1]: create a new folder "types" and add types.ts, interface.ts & enums.ts. And export the enums from enum.ts

File: src/config/constants.config.ts
Comment (arpesh28) [inline src/config/constants.config.ts:1]: You don't need the constants if you're using enums

**Review by arpesh28 — CHANGES_REQUESTED** [review]

**Comment by arpesh28** [comment]
> Personal Tasks CRUD created

Please mention and tag the issue number in here.

File: src/config/zodSchema.config.ts
Comment (omkarvaigankar8) [reply src/config/zodSchema.config.ts:1]: added required types

File: src/config/constants.config.ts
Comment (omkarvaigankar8) [reply src/config/constants.config.ts:1]: removed the constant

**Review by arpesh28 — APPROVED** [review]

**Review by arpesh28 — APPROVED** [review]
Awesome @omkarvaigankar8 

// bounty $50

**Pull request merged**